cms/static/js/views/pages/group_configurations.js

```diff
@@ -26,7 +25,7 @@ function($, _, gettext, BasePage, GroupConfigurationsListView, PartitionGroupLis
                 this.allGroupViewList.push(
                     new PartitionGroupListView({
                         collection: this.allGroupConfigurations[i].get('groups'),
-                        restrictEditing: currentScheme === enrollmentScheme,
+                        restrictEditing: currentScheme === 'enrollment_track' || currentScheme === 'content_type_gate',
```

Comment:
Any easy way we can key this off the `read_only` attribute from the backend?

Reply:
Yeah -- I looked into that. Can do, for sure, however, it would entail sending a different JSON representation of the data to the front-end than is used everywhere else (OLX, POST request bodies etc).

Not sure which one better